app/src/main/java/com/codesoom/assignment/dto/ErrorResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.codesoom.assignment.dto;
+
+/**
+ * 예외 메세지를 담는 객체
+* */
```

Comment:
예외와 에러는 다릅니다. 어떻게 다른지도 조사해 보세요.
```suggestion
/**
 * 에러 메세지.
*/
```